CMSgov/easi-app
# [NOREF] Task list decision step updates

# NOREF

## Changes and Description

These changes were discussed with Zoe when going over updates related to the UX doc item "Closed sub statuses occasionally displaying incorrectly" ([see Slack thread](https://cmsgov.slack.com/archives/CNU2B59UH/p1703001116902059))
- Updated decision banner on task list to only show if request is either closed with a decision issued or open and in the decision step
- Updated decision step link text from "View existing decision" to "View previous decision"
- Unit test updates for closed state alerts



## How to test this change

- Task list decision alert should render for request that is:
  - closed with decision
  - open and in decision step (issue decision and then reopen request to view this state)
- Task list decision alert should NOT render for request that is:
  - open and NOT in decision step
  - closed with no decision issued
- Decision step link text should say "View previous decision" if request has decision issued but was reopened and moved to previous step



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
